slides/all-templates/index.ftd

```diff
@@ -0,0 +1,137 @@
+
```

Comment:
What is this? If this is a "page" it must be in `pages`. If it is a component it must be in `components`. 

Also what is the different between `all-templates/index.ftd` and `slides/pages/templates.ftd`? We already have an all templates page, which is called `pages/templates`.